ddtrace/contrib/kafka/patch.py

```diff
@@ -71,6 +79,7 @@ def patch():
 
     trace_utils.wrap(TracedProducer, "produce", traced_produce)
     trace_utils.wrap(TracedConsumer, "poll", traced_poll)
+    trace_utils.wrap(TracedConsumer, "commit", traced_commit)
```

Comment:
Why are we wrapping the tracedconsumer? Can we move logic from `traced_commit` to `TracedConsumer.commit`

Reply:
How is that different from `poll` or `produce` ?
I'm not sure I understand your suggestion.

Reply:
I guess this suggestion would also apply to poll or produce. There's no need to refactor this code right now. I think the current approach is fine.